src/invidious/views/user/preferences.ecr

```diff
@@ -284,9 +284,18 @@
 
                 <div class="pure-control-group">
                     <label for="popular_enabled"><%= translate(locale, "Popular enabled: ") %></label>
-                    <input name="popular_enabled" id="popular_enabled" type="checkbox" <% if CONFIG.popular_enabled %>checked<% end %>>
+                    <input name="popular_enabled" id="popular_enabled" type="checkbox" <% if CONFIG.page_enabled?("popular") %>checked<% end %>>
                 </div>
 
+                <div class="pure-control-group">
+                    <label for="trending_enabled"><%= translate(locale, "Trending enabled: ") %></label>
+                    <input name="trending_enabled" id="trending_enabled" type="checkbox" <% if CONFIG.page_enabled?("trending") %>checked<% end %>>
+                </div>
+
+                <div class="pure-control-group">
+                    <label for="search_enabled"><%= translate(locale, "Search enabled: ") %></label>
+                    <input name="search_enabled" id="search_enabled" type="checkbox" <% if CONFIG.page_enabled?("search") %>checked<% end %>>
+                </div>
```

Comment:
We are trying to unify the naming/format of translation keys (the string that is being searched to get the associated translated string):

```suggestion
                <div class="pure-control-group">
                    <label for="trending_enabled"><%= translate(locale, "preferences_trending_enabled_label") %></label>
                    <input name="trending_enabled" id="trending_enabled" type="checkbox" <% if CONFIG.page_enabled?("trending") %>checked<% end %>>
                </div>

                <div class="pure-control-group">
                    <label for="search_enabled"><%= translate(locale, "preferences_search_enabled_label") %></label>
                    <input name="search_enabled" id="search_enabled" type="checkbox" <% if CONFIG.page_enabled?("search") %>checked<% end %>>
                </div>
```

In all cases, you need to add these key/value combinations to the end of `locales/en_US.json` for them to be translated in other languages:

```json
{
    ...
    "some_other_key": "Some other string",
    "preferences_trending_enabled_label": "Trending enabled: ",
    "preferences_search_enabled_label": "Search enabled: "
}
```